python/tk_multi_publish2/dialog.py

```diff
@@ -1288,29 +1289,20 @@ def _publish_task_generator(self):
             try:
                 # yield each child item to be acted on by the publish api
                 if isinstance(ui_item, TreeNodeTask):
-                    pub_exception = yield ui_item.task
+                    yield ui_item.task
```

Comment:
there was already a catch all exception handler below these lines, so now the exception bubbles up to the GUI and is displayed.